rasa/core/schemas/domain.yml

```diff
@@ -7,6 +7,8 @@ mapping:
       mapping:
         use_entities:
           type: "bool"
```

Comment:
both `use_entities` and `ignore_entities` can also be lists, can't they?

Reply:
I couldn't find a type for list (the ones that look like `[...]`). I guess we could use a regex, but felt that would be overly complicated. This is the list of types they offer - https://pykwalify.readthedocs.io/en/unstable/validation-rules.html

Reply:
Mhm, should be `sequence` or `seq`

Reply:
Yeah, that's what I meant by "their list is a multiline thing". This is what they mean.
```md
# Data
- 'Foobar'
- 'Barfoo'
```

Reply:
thats the same thing :-) https://stackoverflow.com/questions/33989612/yaml-equivalent-of-array-of-objects-in-json

Reply:
I ran pykwalify with this schema
```yaml
allowempty: True
mapping:
  intents:
    type: "seq"
    sequence:
    - type: "map"
      mapping:
        use_entities:
          type: "seq"
...
```
on this file
```yaml
intents:
  - greet
  - goodbye
  - affirm
  - deny
  - mood_great:
      use_entities: [1, 2, 3]
  - mood_unhappy
```
and got this error:
`pykwalify.errors.SchemaConflict: <SchemaConflict: error code 5: Type is sequence but no sequence alias found on same level: Path: '/mapping/intents/sequence/0/mapping/use_entities'>`

Reply:
Ah, I guess that's settled then!

But we have a more fundamental problem - which is not being able to accommodate multiple data types. We could either do an `any` or choose any ONE single type. Can't do `OR` style types as far as what I read.

Reply:
Thought the custom validators might help, but according to [this](https://pykwalify.readthedocs.io/en/unstable/extensions.html), it appears you can only add more failure cases for the parsing (and not do things that'd make it more lenient).

Reply:
mhm, that's a shame. Thanks for digging into this! let's put `any` then :-)